sentry-android-core/src/main/java/io/sentry/android/core/ContextUtils.java

```diff
@@ -28,16 +30,46 @@ static PackageInfo getPackageInfo(final @NotNull Context context, final @NotNull
    * @return the Application's PackageInfo if possible, or null
    */
   @Nullable
+  @SuppressWarnings("deprecation")
   static PackageInfo getPackageInfo(
       final @NotNull Context context, final int flags, final @NotNull ILogger logger) {
     try {
-      return context.getPackageManager().getPackageInfo(context.getPackageName(), flags);
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
```

Comment:
There's a class that replaces `Build.VERSION` in the SKD to make it testable.

Reply:
@marandaneto I created an istance of `BuildInfoProvider` inside these methods. 
I couldn't pass directly the instance we create at the beginning of the init flow.
Is it fine, since it wasn't created anywhere else?